src/libraries/System.Data.Common/src/System/Xml/DataPointer.cs

```diff
@@ -64,10 +61,10 @@ private DataRow Row
         {
             get
             {
-                XmlBoundElement rowElem = GetRowElement();
+                XmlBoundElement? rowElem = GetRowElement();
                 if (rowElem == null)
                 {
-                    return null;
+                    return null!;
                 }
 
                 Debug.Assert(rowElem.Row != null);
```

Comment:
IN line 75, `node` should be marked nullable.

Reply:
Fixed